src/permutat.c

```diff
@@ -1703,7 +1708,12 @@ Obj             PowPerm4Int (
 Obj InvPerm (
     Obj             perm )
 {
-    return POW( perm, INTOBJ_INT(-1) );
+  Obj inv = STOREDINV_PERM(perm);
+  if (inv != 0)
+    return inv;
+  inv =  POW( perm, INTOBJ_INT(-1) );
```

Comment:
remove extra space before `POW` -- or even run `clang-format` on this function? (resp. use `git clang-format` on the commit? possibly followed by `git add -p` and `git checkout -p` to only keep parts of what `git clang-format` did)